src/api/routers/participants/participantsRouter.ts

```diff
@@ -136,8 +140,10 @@ export function createParticipantsRouter() {
   participantsRouter.get('/:participantId/auditTrail', handleAuditTrail);
 
   participantsRouter.get('/:participantId/users', handleGetParticipantUsers);
+  participantsRouter.get('/:participantId/:userId', handleGetUserRolesForCurrentParticipant);
   const participantUsersRouter = createParticipantUsersRouter();
   participantsRouter.use('/:participantId/users', participantUsersRouter);
+  participantsRouter.use('/:participantId/:userId', participantUsersRouter);
```

Comment:
I don't think this line is required - it implies that the request is handled by the `participantUsersRouter` but it is handled inside `participantUsers.ts`. 

If we follow the API design I [commented ](https://github.com/IABTechLab/uid2-self-serve-portal/pull/557/files#r1818358705)just above, then line 145 should be sufficient to make it route properly.